### PR TITLE
Fix sighting card in single cryptid sightings page

### PR DESCRIPTION
**Pull request by krdurrett**
____ Wrote Tests ___x_ Implemented ____ Reviewed

## Neccesary checkmarks:

    [] All Tests are Passing in all environments

    [x] The code will run locally

## Type of change

    [] New feature
    [x] Bug Fix
    [] Refactor

## Description of Change:

    description: Fix the bug in single cryptid sightings page to allow you to click in to a single sighting from that page
    
## Requested feedback: 
    
    I'd like feedback on... n/a

## Check the correct boxes

    [x] This broke nothing
    [] This broke some stuff
    [] This broke everything

## Testing Changes

    [x] No Tests have been changed
    [] Some Tests have been changed
    [] All of the Tests have been changed(Please describe what in the world happened)

## Checklist:

    [] My code has no unused/commented out code
    [] My code has no binding.pry's
    [] I have reviewed my code
    [] I have commented my code, particularly in hard-to-understand areas
    [] I have fully tested my code

(For Fun!)Please Include a link to a gif of your feelings about this branch

Link:

### CODE REVIEW

**Review by RosaTheDev — APPROVED** [review]
good jjob!

File: src/Components/SingleCryptid.js [hunk 1]
```diff
@@ -30,8 +30,6 @@ const SingleCryptid = () => {
     }
   })
 
-  console.log(loading, data, error)
```
Comment (GraceGardner) [inline src/Components/SingleCryptid.js:33]: Good catch finding all of the console.logs and getting those out of there.

File: src/Components/SingleCryptidSightings.js [hunk 2]
```diff
@@ -39,12 +38,14 @@ const SingleCryptidSightings = () => {
...
     return setError(error)
   }
-  console.log(data)
 
   const sightingCards = data.cryptidByName[0].sightings.map(sighting => {
+    console.log(sighting)
     return (
       <SightingCard
         key={sighting.id}
+        id={sighting.id}
```
Comment (GraceGardner) [inline src/Components/SingleCryptidSightings.js:47]: Good catch on this. Glad that the errors helped us figure this out.

File: src/Components/SingleCryptidSightings.js [hunk 3]
```diff
@@ -58,28 +59,6 @@ const SingleCryptidSightings = () => {
     <>
       <div className='sightings-header'>
         <h1 className='sightings-text'>{cryptid} Sightings</h1>
-        {/* <div className='search-container'>
```
Comment (GraceGardner) [inline src/Components/SingleCryptidSightings.js:61]: Thanks for removing all of this.

**Review by GraceGardner — APPROVED** [review]
Good work on this bug fix. Glad you found this.

**Review by RosaTheDev — APPROVED** [review]

**Pull request merged**